apps/myaccount/src/types/imports.d.ts

```diff
@@ -32,3 +32,10 @@ declare module "*.png" {
     const content: string;
     export default content;
 }
+
+declare module "worker-loader!*" {
```

Comment:
Is this necessary since we are now using the Webpack config to load the web worker?

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/1994/commits/988c77fc9713b7e4947f05dbc7c4c993be06cc71